packages/clay-pagination/src/Pagination.tsx

```diff
@@ -10,24 +10,45 @@ import Ellipsis from './Ellipsis';
 import Item from './Item';
 
 interface IProps extends React.HTMLAttributes<HTMLUListElement> {
+	/**
+	 * Use this property for defining `otherProps` that will be passed to the `nav` element.
+	 */
+	navProps?: {
+		'aria-label'?: string;
+		role?: string;
+	};
+
 	/**
 	 * The size of pagination element.
 	 */
 	size?: 'lg' | 'sm';
 }
 
 const ClayPagination = React.forwardRef<HTMLUListElement, IProps>(
-	({children, className, size, ...otherProps}: IProps, ref) => {
+	(
+		{children, className, navProps = {}, size, ...otherProps}: IProps,
+		ref
+	) => {
 		return (
-			<ul
-				{...otherProps}
-				className={classNames('pagination pagination-root', className, {
-					[`pagination-${size}`]: size,
-				})}
-				ref={ref}
+			<nav
+				aria-label={navProps['aria-label'] || 'Pagination'}
+				role={navProps?.role || 'navigation'}
```

Comment:
I'm not sure if this should be configurable, I guess because it's a Pagination component it would have another option besides `navigation`?